### PR TITLE
Add minor fixes for 22.05.

### PR DESCRIPTION
**Pull request by kwitaszczyk**

### CODE REVIEW

File: src/background/README.md [hunk 1]
```diff
@@ -44,6 +44,19 @@ all language- and sub-language pointers using CHERI capabilities rather than
 architectural integers.
 With suitable OS support, they can also support temporal memory safety.
 
+CHERI C/C++ support is implemented by [CHERI
+LLVM](https://www.cl.cam.ac.uk/research/security/ctsrd/cheri/cheri-llvm.html).
+The Clang/LLVM compiler is able to statically identify and report as warnings
+many incompatibilites between C/C++ and CHERI C/C++ that would result in
+run-time CHERI exceptions. However, a correctly compiled CHERI C/C++ program
+might still trigger CHERI capability violations in run-time that would have to
```
Comment (rwatson) [inline src/background/README.md:52]: Maybe “that can be”?

File: src/support/README.md [hunk 2]
```diff
@@ -24,7 +24,8 @@ feedback and improvements would be very much appreciated.
 Support for CheriBSD is provided via the [CHERI CPU
 Slack](https://www.cl.cam.ac.uk/research/security/ctsrd/cheri/cheri-slack.html),
 where there is a `#cheribsd` channel where many of CheriBSD developers and
-users can be found.
+users can be found and a `#software-porting` channel where discussions on
+software porting to CHERI are held.
```
Comment (rwatson) [inline src/support/README.md:28]: Maybe we should start making this a structured list of some sort, identifying and briefly describing a few key channels?

File: src/background/README.md (hunk 1 above)
Comment (kwitaszczyk) [reply src/background/README.md]: OK.

File: src/support/README.md (hunk 2 above)
Comment (kwitaszczyk) [reply src/support/README.md]: OK.

File: src/background/README.md [hunk 3]
```diff
@@ -44,6 +44,19 @@ all language- and sub-language pointers using CHERI capabilities rather than
...
+The Clang/LLVM compiler is able to statically identify and report as warnings
+many incompatibilites between C/C++ and CHERI C/C++ that would result in
+run-time CHERI exceptions. However, a correctly compiled CHERI C/C++ program
+might still trigger CHERI capability violations in run-time that can be
+diagnosed with
+[GDB](https://www.cl.cam.ac.uk/research/security/ctsrd/cheri/cheri-software.html).
+
+You can learn more about CHERI C/C++ and its CheriABI run-time environment by
+reading the technical report, [CHERI C/C++ Programming
+Guide](https://www.cl.cam.ac.uk/techreports/UCAM-CL-TR-947.pdf).
```
Comment (rwatson) [inline src/background/README.md:58]: I guess we should also link to this in Resources [if we don’t already].

Comment (kwitaszczyk) [reply src/background/README.md:58]: The guide is already listed in Resources.

**Review by rwatson — APPROVED** [review]

**Pull request merged**